octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/util/error/ErrorComposite.java

```diff
@@ -86,90 +91,106 @@ public void widgetSelected(SelectionEvent e) {
     }
-    
+
     public void displayException(Exception ex) {
-        if(ex instanceof OctaneException) {
-            displayOctaneException((OctaneException)ex);
+        if (ex instanceof OctaneException) {
+            displayOctaneException((OctaneException) ex);
         } else {
-            displayGenericException(ex); 
+            displayGenericException(ex);
         }
     }
-    
-    private void displayGenericException(Exception ex) {        
+
+    private void displayGenericException(Exception ex) {
         compositeExceptionData.setLayout(new GridLayout(1, false));
-        
+
         TruncatingStyledText txtErrorField = new TruncatingStyledText(compositeExceptionData, SWT.MULTI);
         txtErrorField.setForeground(SWTResourceManager.getColor(SWT.COLOR_DARK_RED));
         txtErrorField.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, true, 1, 1));
-        
+
         txtErrorField.setText(ex.getMessage());
     }
-    
+
     private void displayOctaneException(OctaneException ex) {
         compositeExceptionData.setLayout(new GridLayout(2, false));
-        
+
         ErrorModel errorModel = ex.getError();
         String description = getDescriptionFromOctaneException(errorModel);
         
-        if(description != null) {
-            TruncatingStyledText txtErrorField = new TruncatingStyledText(compositeExceptionData, SWT.NONE);
+
+        if (description != null) {
+            TruncatingStyledText txtErrorField = new TruncatingStyledText(compositeExceptionData, SWT.WRAP | SWT.MULTI | SWT.READ_ONLY);
             txtErrorField.setForeground(SWTResourceManager.getColor(SWT.COLOR_DARK_RED));
             txtErrorField.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 2, 1));
             txtErrorField.setText(description);
-            
+
             errorModel.removeValue(ERORR_MODEL_FIELD_DESCRIPTION);
             errorModel.removeValue(ERORR_MODEL_FIELD_DESCRIPTION_TRANSLATED);
         }
-        
-        //Rest of values displayed generically
+
+        // Rest of values displayed generically
         ex.getError().getValues().forEach(fieldModel -> {
-            
             String fieldValueTxt = getFieldModelValueIfPresentAndNotEmpty(errorModel, fieldModel.getName());
-            
-            if(fieldValueTxt == null) {
+            if (fieldValueTxt == null) {
                 return;
             }
-            
-            Label lblErrorField = new Label(compositeExceptionData, SWT.NONE);
-            lblErrorField.setText(convertFieldNameToLabel(fieldModel.getName()));
-            
-            TruncatingStyledText txtErrorFieldValue = new TruncatingStyledText(compositeExceptionData, SWT.NONE);
-            txtErrorFieldValue.setLayoutData( new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
-            txtErrorFieldValue.setText(fieldValueTxt);
+            if (fieldModel.getName().equals(ERROR_MODEL_FIELD_STACK_TRACE)) {
```

Comment:
Wouldn't it be easier to use the remove field method I used for description, then you don't need to declare a field either

Reply:
@atothhpe done.ty